### ISSUE TITLE
Intro Slider

### ISSUE THREAD

**Issue by himanshusharma89** [himanshusharma89/Flutter-Blog-App#24]
A full-screen intro slider when user freshly installs the app before registration.

**Comment by hutomosaktikartiko** [comment]
> A full-screen intro slider when user freshly installs the app before registration.

do you already have a design that will be applied to the code?

**Comment by himanshusharma89** [comment]
@hutomosaktikartiko I don't have a particular design, you can follow up the current UI and create the intro slider

**Comment by himanshusharma89** [comment]
Hi @Dhruv-Sachdev1313, you can create PageView with 3 Pages for intro slider with the help of smooth page indicator when the user freshly installs the app.

**Comment by Dhruv-Sachdev1313** [comment]
heyy @himanshusharma89 , Thanks for tagging me in this issue. Anything specific you want in intro sliders?

**Comment by himanshusharma89** [comment]
@Dhruv-Sachdev1313 Just create a page view slider, with appropriate assets for the App with current UI in mind. you can refer to https://github.com/smaranjitghose/girlscript_app/issues/270#issuecomment-703073809

**Comment by Dhruv-Sachdev1313** [comment]
@himanshusharma89 ohkayy . I am on it . please assign it to me

**Comment by Dhruv-Sachdev1313** [comment]
heyy @himanshusharma89 I know this is a very naive question but I couldn't make your code run in my machine. I guess it is the lowercase issue of the name of the project. Can you please guide ?
<img width="1440" alt="Screenshot 2020-10-06 at 1 48 47 AM" src="https://user-images.githubusercontent.com/56223242/95128117-97d84300-0776-11eb-9762-749171160c4c.png">

**Comment by himanshusharma89** [comment]
@Dhruv-Sachdev1313 No need to create a flutter project, pull the latest from the remote. Do the pub get in `pubspec.yaml` and restart vs code.

**Comment by Dhruv-Sachdev1313** [comment]
heyy @himanshusharma89 it worked.  I just have a question, Is firebase setup for IOS , I mean have you added the googleService.info.plist file in runner directory? because I can't see any

**Comment by himanshusharma89** [comment]
Hi @Dhruv-Sachdev1313, I don't have Xcode so I couldn't integrate it for iOS. Share me your email, I'll add you to the console.

**Comment by Dhruv-Sachdev1313** [comment]
@himanshusharma89 my email id is dhruvhsachdev@gmail.com, or else I could get an android phone to test this app. anything is fine for me.

**Comment by himanshusharma89** [comment]
@Dhruv-Sachdev1313 Added to console. And I think we should go for iOS integration. Create an issue for iOS integration, I'll assign it to you.

**Comment by himanshusharma89** [comment]
@Dhruv-Sachdev1313 Any update?

**Comment by krish-bhanushali** [comment]
Can I work on this issue?

**Comment by Dhruv-Sachdev1313** [comment]
@himanshusharma89  Yess , I wanted to add gif like image, so I am trying to figure out how could a record a screen video in my simulator and convert it to gif, and also I am working on completing the firebase for iOS setup, just give me a little time if its ok for you ?

**Comment by himanshusharma89** [comment]
Sure @Dhruv-Sachdev1313, take your time and if you need gifs, checkout: https://lottiefiles.com/

**Comment by Dhruv-Sachdev1313** [comment]
@himanshusharma89 Hey, is it ok if I make PR till this weekend? actually I am really occupied with college stuff. Am I making the project delay?

**Comment by himanshusharma89** [comment]
@Dhruv-Sachdev1313 It is okay, take your time.

**Comment by Dhruv-Sachdev1313** [comment]
Hey @himanshusharma89 . I am facing a little trouble with choosing the colour part, Could you please give me some reviews and suggestions, as you can say that I am not that  good when it comes to choosing colour combinations😅. By the way in the last screenshot for dark mode that image is a GIF for dark mode, I also have uploaded that gif at the last .

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-16 at 19 52 13](https://user-images.githubusercontent.com/56223242/96270623-864e3100-0fe9-11eb-9703-0958e36d8e6e.png)
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-16 at 19 52 16](https://user-images.githubusercontent.com/56223242/96270628-8817f480-0fe9-11eb-81fe-f0740093b387.png)
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-16 at 19 52 25](https://user-images.githubusercontent.com/56223242/96270635-8a7a4e80-0fe9-11eb-9bcb-066f5e801899.png)

![darkmode](https://user-images.githubusercontent.com/56223242/96270760-b09fee80-0fe9-11eb-88d2-e5225914a20f.gif)

**Comment by Dhruv-Sachdev1313** [comment]
![darkmode](https://user-images.githubusercontent.com/56223242/96270760-b09fee80-0fe9-11eb-88d2-e5225914a20f.gif)

**Comment by himanshusharma89** [comment]
@Dhruv-Sachdev1313 Use rounded icons and normal text Alignment. And about the colour, I am not so good at choosing the colour, you can refer to https://coolors.co/ to generate a palette

**Comment by Dhruv-Sachdev1313** [comment]
@himanshusharma89 Thanks for the colour website, but I couldn't understand exactly what do you mean by the rounded icons and normal text alignment, can you please explain a little more about that

**Comment by himanshusharma89** [comment]
@Dhruv-Sachdev1313 In the beta branch you have updated icons check them and don't make the text italic. This was what I meant.

**Comment by Dhruv-Sachdev1313** [comment]
@himanshusharma89 By beta branch you mean the beta channel of flutter sdk ???

**Comment by himanshusharma89** [comment]
Yeah @Dhruv-Sachdev1313

**Comment by Dhruv-Sachdev1313** [comment]
@himanshusharma89 . Hey I was searching about the iOS SDK issue in firebase that we faced, I saw it on Youtube, that guy faced the same issue. I guess that it is a bug from firebase side, because as you saw even that day that our app was communicating with firebase servers , but it did not update the status in console.

**Comment by himanshusharma89** [comment]
@Dhruv-Sachdev1313 Some times Android also face this issue, you can try it again, it should work and what about the intro slider? Any update?

**Comment by Dhruv-Sachdev1313** [comment]
@himanshusharma89 Ohkaay, i will try again, About the intro slider, I am still not convinced full by how it looks, just a question as I have shared the screenshot before, is there anything on content side to add or delete?

**Comment by himanshusharma89** [comment]
@Dhruv-Sachdev1313 You can add some illustrations/vectors in the intro slides, I'll add the content later. And pull the code before working on it.

**Comment by Dhruv-Sachdev1313** [comment]
hey @himanshusharma89 . Sorry for the delay in this issue, but I am facing a problem due to which this app is not running in my simulator. This Issue is similar to this one https://github.com/firebase/firebase-ios-sdk/issues/6462
I have added the screenshot of the error, I have tried updating the pods several times , but it didn't work.
<img width="1440" alt="Screenshot 2020-11-11 at 10 46 01 PM" src="https://user-images.githubusercontent.com/56223242/98842741-055f4980-2470-11eb-9b2a-9c35c2289002.png">

**Comment by himanshusharma89** [comment]
Hey @Dhruv-Sachdev1313, sorry for the late response, are you still facing the issue?

**Comment by Saransh-cpp** [comment]
Hey, if the assignee is not able to make it, can I work on this issue? I had some doubts, there is no registration screen so how will the showing intro sliders only once thing work? Should I use SharedPreferences to store the user's data (that if he has already seen the screens or not) locally?

**Comment by himanshusharma89** [comment]
Yeah @Saransh-cpp, SahredPreferences will be used to store the data.

**Comment by Kishankumar-10** [comment]
Hello,
I’m interested in contributing to implement the full-screen intro slider (Issue #24) in the Flutter Blog App. I have experience with Flutter and would be glad to help improve this feature. Please let me know how I can get started.

**Comment by himanshusharma89** [comment]
Sure, go ahead @Kishankumar-10. The app already has it, but if you can improve it, then go ahead. Feel free to raise other issues while working on it as it is outdated
